miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/RealMiniApp.kt

```diff
@@ -47,6 +61,9 @@ internal class RealMiniApp(
         appId.isBlank() -> throw sdkExceptionForInvalidArguments()
         else -> {
             val (basePath, miniAppInfo) = miniAppDownloader.getMiniApp(appId)
+            val versionId = fetchInfo(appId).version.versionId
```

Comment:
I think you don't need to use `fetchInfo` here. You already have `miniAppInfo` from above, so you can just use the `version.versionId` from that.